functions/send_encounters/main.py

```diff
@@ -46,32 +39,27 @@ def send_encounters(request):
     lang = request_data[KEY_LANG]
     encounters = request_data[KEY_ENCOUNTERS]
 
-    user_entity = _get_user_entity(user_id)
-
-    if not user_entity:
-        return jsonify({"status": "failed", "message": "unauthorized"}), 401
+    try:
+        user_entity = get_user_from_datastore(user_id)
+    except InvalidRequestException as e:
+        return jsonify(
```

Comment:
Same error as we had in /send_encounters

Should be `return jsonify(e.response), e.status`

Reply:
Thanks ;) rebase-merge issues... fixed